<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArticleInfoManage</class>
 <widget class="QDialog" name="ArticleInfoManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_Operation">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Delete">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Note">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Insert">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑文章信息 状态：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Reset">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_ArticleContent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Up">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>上移</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Down">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>下移</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文章内容：</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文章标题：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ArticleTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
